src/AccessibilityInsights.SharedUx/Dialogs/ChangeChannelContainedDialog.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.SetupLibrary;
+using System.Globalization;
+using System.Windows;
+
+namespace AccessibilityInsights.SharedUx.Dialogs
+{
+    /// <summary
+    /// Interaction logic for ChangeChannelContainedDialog.xaml
+    /// </summary>
+    public partial class ChangeChannelContainedDialog : ContainedDialog
```

Comment:
how does it handle {ESC} button? typically, we close a dialog when ESC key is pressed as a cancel action. I don't see a code for that.

Reply:
ESC now handled